### DISCUSSION TITLE
Request for ET-4800 support?

### DISCUSSION THREAD

**Discussion by jakusotsu** [Ircama/epson_print_conf#14] [Ideas]
Hi, I got a great deal for ET-4800 and hoping for this script to support it for the future

https://codeberg.org/atufi/reinkpy/src/branch/main/reinkpy/epson.toml#L2622

Thanks!

**Comment by Ircama** [comment]
You can test the related [hint](https://github.com/Ircama/epson_print_conf?tab=readme-ov-file#example-to-integrate-new-printers) in the README.
